rkt/rktflag/bitflags.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rktflag
+
+import (
+	"fmt"
+)
+
+// BitFlags is a flag value type supporting a csv list of options stored as bits
+type BitFlags struct {
+	*OptionList
+	flags   int
+	flagMap map[string]int
+}
+
+func NewBitFlags(permissibleOptions []string, defaultOptions string, flagMap map[string]int) (*BitFlags, error) {
```

Comment:
Ha. Well spotted. No, it won't be imminently used for something. I first made it public and then replaced its uses outside `rktflag` with newly-introduced `NewSecFlags`. I'll hide both `BitFlags` and `NewBitFlags` then.